apps/dashboard_app/helpers/ekubo.py

```diff
@@ -81,14 +90,13 @@ def fetch_liquidity(self, bids: bool = True) -> dict[str, str | list[float]]:
         :return: dict[str, str | list[float]]
         """
 
-        params = self.params_for_bids
+        params = self.params_for_bids if bids else self.params_for_asks
         if not bids:
-            params = self.params_for_asks
             logging.warning(
                 "Using collateral token as base token and debt token as quote token."
             )
 
-        response = requests.get(self.URL, params=params)
+        response = requests.get(self.URL, params=params, timeout=10)
```

Comment:
why did you add this code?

Reply:
Hello there every code I modified was based on suggestion from pylint like error: C0206: Consider iterating with .items()(consider using-duct-items) or warning W3101 about not adding the timeout Argus in the request.get().